include/nlohmann/json_fwd.hpp

```diff
@@ -36,7 +36,8 @@ template<template<typename U, typename V, typename... Args> class ObjectType =
          template<typename U> class AllocatorType = std::allocator,
          template<typename T, typename SFINAE = void> class JSONSerializer =
          adl_serializer,
-         class BinaryType = std::vector<std::uint8_t>>
+         class BinaryType = std::vector<std::uint8_t>, // cppcheck-suppress syntaxError
```

Comment:
Can you please check if this is still needed for cppcheck 2.8? I believe CI is using this version.

Reply:
It is not required anymore. I removed it.